### DISCUSSION TITLE
It seems that the options.add_argument() functions are not working in the following code:

### DISCUSSION THREAD

**Discussion by y0un9kane** [ultrafunkamsterdam/undetected-chromedriver#1188] [General]
It seems that the options.add_argument() functions are not working in the following code:

python
Copy code
import undetected_chromedriver as uc
options = uc.ChromeOptions()

options.add_argument('--window-size=800,600')
options.add_argument('--start-maximized')
options.add_argument('--disable-tracking')

driver = uc.Chrome(
    options=options,
    version_main=112
)
driver.get('https://nowsecure.nl')

**Comment by y0un9kane** [comment]
inoperative
options.add_argument('--window-size=800,600')
options.add_argument('--lang=en-US')
options.add_argument('--disable-tracking')
options.add_argument("--timezone=America/New_York")

**Reply by 007-JB** [reply]
> It seems that the options.add_argument() functions are not working in the following code:
> 
> python Copy code import undetected_chromedriver as uc options = uc.ChromeOptions()
> 
> options.add_argument('--window-size=800,600') options.add_argument('--start-maximized') options.add_argument('--disable-tracking')
> 
> driver = uc.Chrome( options=options, version_main=112 ) driver.get('https://nowsecure.nl')

Working for me....

**Comment by y0un9kane** [comment]
These are certain to work
options.add_argument('--proxy-server=
options.add_argument( '--user-agent=
options.add_argument("--incognito")

**Reply by chibuikeogodo** [reply]
The user agent isn't working, it doesn't change. Please is there anyway else to do it?
